core/src/serialization.rs

```diff
@@ -176,18 +180,21 @@ pub trait Serialize {
 
     /// Bytes of a structure using the CBOR bytes as per the CDDL spec
     /// which for foo = bytes will include the CBOR bytes type/len, etc.
-    fn to_cbor_bytes(&self, force_canonical: bool) -> Vec<u8> {
+    /// This gives the original bytes in the case where this was created
+    /// from bytes originally
+    fn to_original_cbor_bytes(&self) -> Vec<u8> {
         let mut buf = Serializer::new_vec();
```

Comment:
having two functions seemed to be the consensus from the meetings. is this what we want to stay by? it's the most descriptive I think. I'm not sure how much of a deal it'd be space-wise for the JS lib if every single object has an extra function though.

Reply:
should be fine imo